test/lib/npm.js

```diff
@@ -654,3 +655,142 @@ t.test('implicit workspace accept', async t => {
   })
   await t.rejects(mock.npm.exec('org', []), /.*Usage/)
 })
+
+t.test('replaceRegistryHost', async t => {
```

Comment:
Do we have any other tests like this that are testing the internals of config?  These feel overly wrought and potentially unneeded.  We've defined the config. If we want to test the effects of setting these configs it should be tested in a command that would use the setting.

Reply:
I wanted to know for sure that a config option was making it all the way to Arborist in the proper way. Writing these tests actually did prompt me to make a better definition. But yeah, now they're kind of vestigial

Reply:
removed